record_ac/handler.py

```diff
@@ -65,17 +73,17 @@ def add_ac_contact(config, email, first_name='', last_name='', tags=''):
     s = requests.Session()
     resp = s.send(prepared)
 
-    print "Status_code: {}\nContent: {}".format(resp.status_code, resp.content)
+    print ("Status_code: {}\nContent: {}".format(resp.status_code, resp.content))
 
     result = resp.json()
 
     if resp.status_code == 200 and result['result_code'] != 0:
-        print "SUCCESS"
+        print ("SUCCESS")
     else:
         failure_reason = (
             'Failed to create AC contact for {}: {} (status code: {})'.format(
                 email, resp.text, resp.status_code))
-        print failure_reason
+        print (failure_reason)  
```

Comment:
Minor nit-pick - looks like whitespace at the end of the line

Reply:
Fixed